features/aanschrijfwijze.feature

```diff
@@ -553,7 +553,7 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | fields              | naam.aanschrijfwijze            |
       Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
       | naam                  | waarde            |
-      | gebruikinlopendetekst | <aanschrijfwijze> |
+      | aanschrijfwijze.naam  | <aanschrijfwijze> |
```

Comment:
ben je hier vergeten <aanschrijfwijze> te hernoemen naar <naam in aanschrijfwijze>?

Reply:
Ja, aangepast